### ISSUE TITLE
Enhancement: In-Loop Replacement

### ISSUE THREAD

**Issue by mboyas-mitre** [verbal-autopsy-software/InterVA4#3]
https://github.com/verbal-autopsy-software/InterVA4/blob/792fd94fcad1f143f55865e580a4ea0bc233e5bb/InterVA4_1.7/R/InterVA.R#L288-291

These lines are converting categorical to binary on a line-by-line basis.  It might be easier (and likely more efficient) to move this operation outside of the `for` loop and convert the entire input dataset at once.

**Comment by richardli** [comment]
agree, only doing this because of replication from the InterVA software.
